src/operator-docs/filtering/takeWhile.ts

```diff
@@ -1,6 +1,66 @@
 import { OperatorDoc } from '../operator.model';
 
 export const takeWhile: OperatorDoc = {
-  'name': 'takeWhile',
-  'operatorType': 'filtering'
+  name: 'takeWhile',
+  operatorType: 'filtering',
+  signature:
+    'public takeWhile(predicate: function(value: T, index: number): boolean): Observable<T>',
```

Comment:
@btroncone suggested to remove the generic from the signature

Reply:
@JWO719 @btroncone  - Generic on the return type ? Just so I am on the page.

Reply:
if i understand @btroncone at #228 correctly, yes

Reply:
@ashwin-sureshkumar @JWO719 Yes, let's leave out in signatures per https://github.com/ReactiveX/rxjs-docs/pull/196#discussion_r157231328